charts/scalar-manager/templates/_helpers.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
```

Comment:
This helper is generated by `helm create`.
I removed some unused definition though.